chore:  clarify line params docs (#243)
getnacelle/nacelle-js: packages/shopify-cart/README.md

_Creates a new Shopify cart._

> **Accepts**: params - an optional object containing the following optional values: `attributes` (`array`), `buyerIdentity` (`object`), `discountCodes` (`array`), `lines` (`array`), `note` (`string`). To initialize an empty cart, exclude all parameters. For more information on the cart optional values, checkout Shopify's [`CartInput`][shopify-cart-input]. Note that our lines parameter accepts a different value than Shopify's by accepting `nacelleEntryId`.

Reviewer: I think this could be reworded to improve clarity:

```suggestion
**Accepts**: params - an optional object containing the following optional values: `attributes` (`array`), `buyerIdentity` (`object`), `discountCodes` (`array`), `lines` (`array`), `note` (`string`). To initialize an empty cart, exclude all parameters. For more information on the cart optional values, checkout Shopify's [`CartInput`][shopify-cart-input]. Note that the `lines` parameter differs from Shopify [`CartLineInput`][shopify-cart-line-input] in that it expects the variant's `nacelleEntryId` instead of a Shopify `merchandiseId`.
```

> **Accepts**: params - an optional object containing the following optional values: `attributes` (`array`), `buyerIdentity` (`object`), `discountCodes` (`array`), `lines` (`array`), `note` (`string`). To initialize an empty cart, exclude all parameters. For more information on the cart optional values, checkout Shopify's [`CartInput`][shopify-cart-input].Note that the `lines` parameter differs from Shopify [`CartLineInput`][shopify-cart-line-input] in that it expects the variant's `nacelleEntryId` instead of a Shopify `merchandiseId`.

Reviewer: Missing space:

```suggestion
**Accepts**: params - an optional object containing the following optional values: `attributes` (`array`), `buyerIdentity` (`object`), `discountCodes` (`array`), `lines` (`array`), `note` (`string`). To initialize an empty cart, exclude all parameters. For more information on the cart optional values, checkout Shopify's [`CartInput`][shopify-cart-input]. Note that the `lines` parameter differs from Shopify [`CartLineInput`][shopify-cart-line-input] in that it expects the variant's `nacelleEntryId` instead of a Shopify `merchandiseId`.
```

_Adds lines to an existing Shopify cart._

> **Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to add. For more information on the cart `lines`, checkout Shopify's [`CartLineInput`][shopify-cart-line-input]. Note that our lines parameter accepts a different value than Shopify's by accepting `nacelleEntryId`.

Reviewer: I think this could be reworded to improve clarity:

```suggestion
**Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to add. For more information on the cart `lines`, checkout Shopify's [`CartLineInput`][shopify-cart-line-input].  Note that the `lines` parameter differs from Shopify [`CartLineInput`][shopify-cart-line-input] in that it expects the variant's `nacelleEntryId` instead of a Shopify `merchandiseId`.
```

_Updates lines on an existing Shopify cart._

> **Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to update. For more information on the cart `lines`, checkout Shopify's [`CartLineUpdateInput`][shopify-cart-line-update-input]. Note that our lines parameter accepts a different value than Shopify's by accepting `nacelleEntryId`.

Reviewer: I think this could be reworded to improve clarity:

```suggestion
**Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to update. For more information on the cart `lines`, checkout Shopify's [`CartLineUpdateInput`][shopify-cart-line-update-input].  Note that the `lines` parameter differs from Shopify [`CartLineInput`][shopify-cart-line-input] in that it expects the variant's `nacelleEntryId` instead of a Shopify `merchandiseId`.
```

  cartId: 'gid://shopify/Cart/112233',
  lines: [
    {

>       cartLineId: 'gid://shopify/CartLine/e543caf96fa645b54d07ab2035ef9dba?cart=a7aad2fb1e6611422c946f6827710550'

Reviewer: `cartLineId` is not a property of `NacelleCartLineItemInput`.

```suggestion
```

Author: > We might also want to adjust the lines where we link out to Shopify since our inputs no longer 1:1 match Shopify's
> 
> eg:
> 
> > For more information on the cart optional values, checkout Shopify's [`CartInput`][shopify-cart-input]

Good catch @mdarrik thank you!

Reviewer: @KrisQ thanks for updating the method name. But the suggestion still stands - `cartLineId` is not a property of `lines` (`NacelleCartLineItemUpdateInput`) in `cartLinesUpdate` either:

![cartLinesUpdate line params id attributes nacelleEntryId quantity sellingPlanId](https://user-images.githubusercontent.com/5732000/191350061-88e45943-0b02-4361-85aa-9862e84ff234.png)

Author: Apologies for the confusion, It should be `id` you're right

Reviewer: No worries at all!
